pkg/collector/corechecks/cluster/kubernetesapiserver/events_common_test.go

```diff
@@ -47,6 +49,59 @@ func TestGetDDAlertType(t *testing.T) {
 	}
 }
 
+func Test_getInvolvedObjectTags(t *testing.T) {
+	taggerInstance := local.NewFakeTagger()
+	taggerInstance.SetTags("namespace://default", "workloadmeta-kubernetes_node", []string{"team:container-int"}, nil, nil, nil)
+	tests := []struct {
+		name           string
+		involvedObject v1.ObjectReference
+		tags           []string
+	}{
+		{
+			name: "get pod basic tags",
+			involvedObject: v1.ObjectReference{
+				Kind:      "Pod",
+				Name:      "my-pod",
+				Namespace: "my-namespace",
+			},
+			tags: []string{
+				"kube_kind:Pod",
+				"kube_name:my-pod",
+				"kubernetes_kind:Pod",
+				"name:my-pod",
+				"kube_namespace:my-namespace",
+				"namespace:my-namespace",
+				"pod_name:my-pod",
+			},
+		},
+		{
+			name: "get pod namespace tags",
+			involvedObject: v1.ObjectReference{
+				Kind:      "Pod",
+				Name:      "my-pod",
+				Namespace: "default",
+			},
+			tags: []string{
+				"kube_kind:Pod",
+				"kube_name:my-pod",
+				"kubernetes_kind:Pod",
+				"name:my-pod",
+				"kube_namespace:default",
+				"namespace:default",
+				"team:container-int",
```

Comment:
nit: I'd add a comment in this one saying that it's the one that comes from the namespace.
Just because the tests are very similar. I think a comment can help to see the difference between the two.